examples/helloworld/Main.cpp

```diff
@@ -14,6 +14,8 @@
 
 #if defined(STM32WBA)
 unsigned int hse_value = 32'000'000;
+#elif defined(STM32H5)
```

Comment:
This default is incorrect. Only the NucleoH563 uses a 8MHz clock. The DiscoveryH573 uses a 25MHz clock

Reply:
The example is specifically for Nucleos, so this value is fine. If someone wants to add different platforms (discovery), then that person may figure out how to distinguish